atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -91,10 +91,11 @@
      * Returns a feign {@link Client} wrapping a {@link okhttp3.OkHttpClient} client with optionally specified {@link SSLSocketFactory}.
      */
     private static Client newOkHttpClient(Optional<SSLSocketFactory> sslSocketFactory) {
-        okhttp3.OkHttpClient client = new okhttp3.OkHttpClient.Builder()
-                .sslSocketFactory(sslSocketFactory.orNull())
-                .build();
-        return new OkHttpClient(client);
+        okhttp3.OkHttpClient.Builder client = new okhttp3.OkHttpClient.Builder();
+        if (sslSocketFactory.isPresent()) {
+            client.sslSocketFactory(sslSocketFactory.get());
```

Comment:
Yeah, but note that https://github.com/square/okhttp/blob/master/okhttp/src/main/java/okhttp3/OkHttpClient.java#L541 doesn't expect an `Optional` but a concrete object.